explorer/cnquery_explorer.proto

```diff
@@ -185,10 +185,12 @@ message QueryPackDocs {
 }
 
 message MqueryDocs {
+  reserved 3;
```

Comment:
do v7 clients require that field being filled in? if so, we should make a deprecated field here with the old type that we can fill in when sending back mquery docs.

Reply:
remediation is not important for the client